### PR TITLE
Fix issues about: multipart/form-data, duration, exampleJsonPath, LRO validation and generate

### PR DESCRIPTION
**Pull request by ankhyk**
- GenerateExamples - Change the domain of LRO header
- ModelValidator - Still validate LRO operation when response doesn't have schema
- ModelValidator - Report real exampleJsonPath when additional parameter includes "."
- Support duration format when generating examples
- SemanticValidator - Fix bug about multipart/form-data

#805 #811 #791 #786

### CODE REVIEW

File: lib/swaggerValidator/semanticValidator.ts [hunk 1]
```diff
@@ -509,13 +510,24 @@ export class SwaggerSemanticValidator {
...
+              NumberOfFileTypeFormData++;
+            }
             if (bodyParam !== undefined) {
               const meta = getOavErrorMeta(
                 param.in === bodyParam.in
                   ? "MULTIPLE_BODY_PARAMETERS"
                   : "INVALID_PARAMETER_COMBINATION",
                 {}
               );
+              if (
```
Comment (raych1) [inline lib/swaggerValidator/semanticValidator.ts:519]: How about the case that have multiple formData parameters but without body parameter?

Comment (raych1) [reply lib/swaggerValidator/semanticValidator.ts:519]: parameters sequence isn't fixed, body param might between multiple form data parameters. So don't need to check on the meta.code. And also assign the bodyParam before continue.

File: test/modelValidatorTests.ts [hunk 2]
```diff
@@ -110,6 +110,15 @@ describe("Model Validation", () => {
         assert.strictEqual(err.innerErrors[0].code, "DOUBLE_FORWARD_SLASHES_IN_URL");
       }
     });
+
+    it("should report real exampleJsonPath when additional parameter includes '.'", async () => {
+      const specPath2 = `${testPath}/modelValidation/swaggers/specification/additionalParameter/additionalParameter.json`;
```
Comment (raych1) [inline test/modelValidatorTests.ts:115]: Is this test for exampleJson path change?

File: test/__snapshots__/exampleGeneratorTests.ts.snap [hunk 3]
```diff
@@ -125,139 +307,4344 @@ Array [
...
-    "examplePosition": Object {
-      "column": 16,
-      "line": 16,
-    },
-    "exampleUrl": "test/exampleGenerator/specification/sql/resource-manager/Microsoft.Sql/preview/2020-02-02-preview/examples/ManagedDatabases_CreateOrUpdate_MaximumSet_Gen.json",
-    "message": "Response statusCode 201 for operation ManagedDatabases_CreateOrUpdate has no response body provided in the example, however the response does have a \\"schema\\" defined in the swagger spec.",
-    "operationId": "ManagedDatabases_CreateOrUpdate",
+    "code": "LRO_RESPONSE_HEADER",
+    "exampleJsonPath": "responses/202/headers",
+    "examplePosition": undefined,
```
Comment (raych1) [inline test/__snapshots__/exampleGeneratorTests.ts.snap:313]: This is regression, position should not be undefined.

File: test/modelValidatorTests.ts (hunk 2 above)
Comment (ankhyk) [reply test/modelValidatorTests.ts:115]: yes, https://devdiv.visualstudio.com/DevDiv/_workitems/edit/1505953/, this is the work item

File: regression/__snapshots__/validateExamplesRegression_latestOnly.test.ts.snap [hunk 4]
```diff
@@ -10122,30 +11257,62 @@ Array [
...
+      "column": 22,
+      "line": 174,
     },
     "schemaUrl": "/home/vsts/work/1/s/regression/azure-rest-api-specs/specification/iotcentral/resource-manager/Microsoft.IoTCentral/stable/2018-09-01/iotcentral.json",
     "severity": 0,
-    "source": "request",
+    "source": "response",
   },
   Object {
-    "code": "PATTERN",
```
Comment (raych1) [inline regression/__snapshots__/validateExamplesRegression_latestOnly.test.ts.snap:10144]: Why this error is gone?

**Review by raych1 — APPROVED** [review]

**Pull request merged**